LoopKit/DeviceManager/DeviceManager.swift

```diff
@@ -53,4 +58,9 @@ public extension DeviceManager {
     var localizedTitle: String {
         return type(of: self).localizedTitle
     }
+    
+    var deviceManagerInstanceIdentifier: String {
```

Comment:
Let's document that this is not unique per instance currently (and this is ok because we currently can only have one instance of each device type), but the intent is provide an attribute that would eventually be unique across the app for all instances.

Reply:
ah... this answers a question I had. Why not remove `Instance` until the time that it is unique across the app? Or will that be happening in the near-future? If not, it will cause confusion until it happens and it seem that this update belongs with that coming code.

Reply:
There's already a lot of confusion about what managerIdentifier is intended for.  A manager identifier identifies a certain kind of device. This is an extension of the api surface to offer an instance identifier that does currently work for all existing use cases. Even though it's currently implemented in a way that does not provide instance identification across multiple instances of a single manager type, it is effective within Loop use as Loop can only have one of each type of device.  When we have the need to support multiple instances of a single type, the implementation can change, and the uses do not need to change.

Reply:
Removed `Instance` and added a comment.